homeassistant/components/tellduslive.py

```diff
@@ -56,7 +57,8 @@ def setup(hass, config):
         return False
 
     hass.data[DOMAIN] = client
-    client.update(utcnow())
+
+    hass.bus.listen(EVENT_HOMEASSISTANT_START, client.update)
```

Comment:
Ok, but with this change, the periodically polling of the server never starts, since update() reschedules itself after each run of sync. Should I make your change, in addition to keeping the hass.bus.listen-stuff?